opentelemetry-api/tests/context/test_context.py

```diff
@@ -74,3 +77,14 @@ def test_set_current(self):
 
         context.detach(token)
         self.assertEqual("yyy", context.get_value("a"))
+
+
+class TestInitContext(unittest.TestCase):
+    def test_load_runtime_context(self):
+        environ[OTEL_PYTHON_CONTEXT] = "contextvars_context"
```

Comment:
nit can you use `patch.dict(environ, ...)` instead to avoid leaking this change outside of the test, and avoid interference from any existing environment variables. Here's an example https://github.com/open-telemetry/opentelemetry-python/blob/47d5ad7aae5aef31238ca66e55dc550b307c7b35/opentelemetry-api/tests/propagators/test_propagators.py#L106-L108

Reply:
Fixed